network-config.yaml

```diff
@@ -9,6 +9,15 @@ live:
     multicall2: "0xcA11bde05977b3631167028862bE2a173976CA11"
     name: Holesky (Infura)
     provider: infura
+  - chainid: 560048
+    # TODO: replace with etherscan explorer when deployed
+    explorer: https://hoodi.cloud.blockscout.com/api/v2
+    host: $HOODI_RPC_URL
+    id: hoodi
+    # New backward-compatible multicall contract. multicall2 is missing on Holesky. See https://github.com/mds1/multicall
+    #multicall2: TODO: deploy multicall on hoodi at put address here
```

Comment:
It's already deployed
https://hoodi.cloud.blockscout.com/address/0xcA11bde05977b3631167028862bE2a173976CA11

Reply:
Thank you for the link! We'll add it to the config